run/test/specs/user_actions.spec.ts

```diff
@@ -188,26 +206,26 @@ async function changeProfilePictureAndroid(platform: SupportedPlatformsType) {
       true
     );
   }
-  await sleepFor(100);
-  await device.clickOnElement(`profile_picture.jpg, 27.75 kB, May 1, 1999`);
+  await device.clickOnElementAll({
+    strategy: "accessibility id",
+    selector: "profile_picture.jpg, 27.75 kB, May 2, 1999",
+  });
   await device.clickOnElementById(
     "network.loki.messenger:id/crop_image_menu_crop"
   );
-  const el = await device.waitForElementToBePresent({
+  const el = await device.waitForTextElementToBePresent({
     strategy: "accessibility id",
     selector: "User settings",
+    maxWait: 10000,
   });
-  await sleepFor(3000);
   const base64 = await device.getElementScreenshot(el.ELEMENT);
   const pixelColor = await parseDataImage(base64);
   console.log("RGB Value of pixel is:", pixelColor);
-  if (pixelColor === "03cbfe") {
+  if (pixelColor === "cbfeff") {
```

Comment:
make this a const (in the function or shared as you have it in a few files) and reuse it.